### PR TITLE
[WIP] Event attendee redirection :tada:

### PR DESCRIPTION
**Pull request by skrix**
### Description

There was a request for such flow that user should be redirected to rules page if clicking an attend button in the email.

### CODE REVIEW

File: app/controllers/visit_requests_controller.rb [hunk 1]
```diff
@@ -24,6 +24,10 @@ def destroy
 
   private
 
+  def user_agreement_redirect
+    redirect_to '/user-agreement'
```
Comment (sveredyuk) [inline app/controllers/visit_requests_controller.rb]: Please, use URL helpers like `user_agreement_path`

**Review by sveredyuk — CHANGES_REQUESTED** [review]

File: app/controllers/visit_requests_controller.rb (hunk 1 above)
Comment (skrix) [reply app/controllers/visit_requests_controller.rb]: dynamically created page, have no path helpers)

Comment (skrix) [reply app/controllers/visit_requests_controller.rb]: but, afm code of conduct should always be created as a static page with the ability to edit content from admin page

**Comment by skrix** [comment]
Registration flow will change another way.

**Pull request closed without merging**